NETWORK/NetworkCreateSynchronisedScene.md

```diff
@@ -7,20 +7,39 @@ ns: NETWORK
 // 0x7CD6BC4C2BBDD526 0xB06FE3FE
 int NETWORK_CREATE_SYNCHRONISED_SCENE(float x, float y, float z, float xRot, float yRot, float zRot, int rotationOrder, BOOL holdLastFrame, BOOL looped, float p9, float animTime, float animSpeed);
 ```
-
 ## Parameters
-* **x**: 
-* **y**: 
-* **z**: 
-* **xRot**: 
-* **yRot**: 
-* **zRot**: 
-* **rotationOrder**: 
-* **holdLastFrame**: 
-* **looped**: 
-* **p9**: 
-* **animTime**: 
-* **animSpeed**: 
+* **x**: The x value of the entity you want to do the animation on.
+* **y**: The y value of the entity you want to do the animation on.
+* **z**: The z value of the entity you want to do the animation on.
+* **xRot**: The xRotation value of the entity you want to do the animation on.
+* **yRot**: The yRotation value of the entity you want to do the animation on.
+* **zRot**: The zRotation value of the entity you want to do the animation on.
+* **rotationOrder**: Rotation order.
+* **holdLastFrame**: Whether the animation ends (smthn like ClearPedTask) if true, or doesn't end till you make it. 
+* **looped**: Whether the animation is looping. 
+* **p9**: a big number like 100000000000 (doesn't seem to affect anything).
```

Comment:
I named this `loopTime` during my analysis, a value that can range from `0.0` to `1.0` according to GTA V's Game Build 323. 

If `holdLastFrame` and `looped` are both set to true and `loopTime` is less than `1.0`, it will be forced to `1.0` regardless of value. But that may never happen since setting `holdLastFrameC` to true would set `loopedC` to `false`.

![image](https://github.com/citizenfx/natives/assets/38232208/3ca93176-a227-4fc2-89ad-ac919170082f)

If `loopTime` is greater than `1.0` it will be forced to `1.0` as well.
![image](https://github.com/citizenfx/natives/assets/38232208/b57753b8-6e19-4f53-8b0a-f4e433aeccf8)